doc/onboarding.md

```diff
@@ -0,0 +1,197 @@
+    * `semver-{minor,major}` labels for gauging versions numbers by commits:
+      * be conservative – that is, if a change has the remote *chance* of breaking something, go for semver-major
+      * when adding a semver label, add a comment explaining why you're adding it
+        * it's cached locally in your brain at that moment!
+
+
+  * Notifying humans
+    * **See "Who to CC in issues"**
+    * will also come more naturally over time
+
+
+  * reviewing:
+    * primary goal is for the codebase to improve
+    * secondary (but not far off) is for the person submitting code to succeed
+      * helps grow the community
+      * and draws new people into the project
+    * Review a bit at a time. It is **very important** to not overwhelm newer people.
+      * it is tempting to micro-optimize / make everything about relative perf,
+        don't succumb to that temptation. we change v8 a lot more often now, contortions
+        that are zippy today may be unnecessary in the future
+    * be aware: your opinion carries a lot of weight!
+    * nits are fine, but try to avoid stalling the PR
+      * note that they are nits when you comment
+      * if they really are stalling nits, fix them yourself on merge (but try to let PR authors know they can fix these)
+      * improvement doesn't have to come all at once
+    * minimum wait for comments time
+      * There is a minimum waiting time which we try to respect for non-trivial changes, so that people who may have important input in such a distributed project are able to respond.
+        * It may help to set time limits and expectations:
+          * the collaborators are very distributed so it is unlikely that they will be looking at stuff the same time as you are.
+          * before merging code: give folks at least one working day to respond: "If no one objects, tomorrow at <time> I'll merge this in."
+            * set reminders
+          * check in on the code every once in a while (set reminders!)
+      * technically, 48 hours for non-trivial changes, and 72 hours on weekends.
+      * if a PR is abandoned, check if they'd mind if you took it over (especially if its just nits left)
+      * you have the power to LGTM another collaborator or TSC / CTC members' work
+
+
+  * what belongs in node:
+    * opinions vary, but I find the following helpful:
+    * if node itself needs it (due to historic reasons), then it belongs in node
+      * that is to say, url is there because of http, freelist is there because of http, et al
+    * also, things that cannot be done outside of core, or only with significant pain (i.e. async-wrap)
+
+
+  * CI testing:
+    * lives here: https://ci.nodejs.org/
+    * make sure to log in – we use github authentication so it should be seamless
+    * go to "node-test-pull-request" and "Build with parameters"
+    * fill in the pull request number without the `#`.
+      * The other options shouldn't need to be adjusted in most cases.
```

Comment:
Mention that they have to check the box indicating that they can vouch that the code is not malicious (or whatever the wording is)